backend/src/main/scala/snorri/Snorri.scala

```diff
@@ -1,11 +1,27 @@
+/*
+ * Copyright 2023 ScalaBridge London
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 package snorri
 
-import krop.all.{*, given}
+import krop.all.*
 
 object Snorri {
-    val route =
-        Route(
-          Request.get(Path.root / "echo" / Param.string),
-          Response.ok[String]
-        ).handle(param => s"Your message was ${param}")
+  val route =
+    Route(
+      Request.get(Path.root / "echo" / Param.string),
+      Response.ok(Entity.text)
```

Comment:
@DmytroMitin this is a test comment to show how to review pull requests.

Reply:
@kannupriyakalra acknowledged, resolving